### ISSUE TITLE
phone number

### ISSUE THREAD

**Issue by fabiovieitas** [Tom60chat/Microsoft-Rewards-Farmer-Sharp#68]
Good afternoon.

You can create account with phone number. But the system does not allow it.

Could you release???

**Comment by Tom60chat** [comment]
What?
Can you redo your sentence?

**Comment by fabiovieitas** [comment]
![image](https://user-images.githubusercontent.com/73303693/214690731-5921f012-d838-4dfa-8ef8-7d221dc2214f.png)


Forgive me. Google translate...

I can create a rewards account with a phone number.

Could you release this function in your script?

**Comment by fabiovieitas** [comment]
![image](https://user-images.githubusercontent.com/73303693/214691052-5372ce05-b48e-4d03-8c52-39542743baa7.png)


Forgive me. Google translate...

I can create a rewards account with a phone number.

Could you release this function in your script?

**Comment by Tom60chat** [comment]
Oh, I see, weird that it doesn't work.

In fact, it was not planned.

Shouldn't be hard to fix.

**Comment by KendallDoesCoding** [comment]
> Oh, I see, weird that it doesn't work.
> 
> In fact, it was not planned.
> 
> Shouldn't be hard to fix.

Yeah. I mean, it can be stated in README.md, that it only works with email address microsoft accounts and NOT phone number accounts?
